src/os/bluestore/BlueStore.cc

```diff
@@ -6742,9 +6759,14 @@ int BlueStore::_setup_block_symlink_or_file(
   return 0;
 }
 
+//#define NCB_FSCK_DEEP_DEBUG
 int BlueStore::mkfs()
 {
   dout(1) << __func__ << " path " << path << dendl;
+#ifdef NCB_FSCK_DEEP_DEBUG
```

Comment:
to be removed?

Reply:
yes, used to debug FSCK with DEEP after mkfs/mount/umount

Reply:
Since only actions performed on NCB_FSCK_DEEP_DEBUG is setting global configuration, it is not necessary to keep it in code.
Remove.